### ISSUE TITLE
Keeps saving changes to the wrong file (loss of data)

### ISSUE THREAD

**Issue by o-alquimista** [lainsce/quilter#106]
I'll describe how I tried to use Quilter.

I had some .odt files that I used with FocusWriter before. Since unfortunately Quilter doesn't read this format, I copied and pasted everything to a new Quilter document and saved with the name "sandbox-notes". Now whenever I try to open new documents and save them, Quilter keeps saving to the same "sandbox-notes" even if I select a totally different location and name for the new file. The resulting "sandbox-notes" becomes either blank (0-byte) or is filled with the text that I intended to save somewhere else.

So, if my new file is named "the-cloud", and I save it to a different location than that of "sandbox-notes", it will still save the changes to "sandbox-notes" (this filename persists at the headerbar).

WORKAROUNDS TRIED:

- deleted WebKitCache at ~/.cache/com.github.lainsce.quilter/

I couldn't find any other relevant data or cache files to delete.

**Comment by lainsce** [comment]
- Which version of Quilter are you using?
- Did you try to open two files at once in Quilter? That's a bad idea since Quilter only opens one file at a time.
- Did you make the new file outside of Quilter? Does this fix the bug?

**Comment by o-alquimista** [comment]
> Which version of Quilter are you using?
Version:

com.github.lainsce.quilter/xenial,now 1.4.3 amd64

> Did you try to open two files at once in Quilter?

No, I intend to open one file at a time.

> Did you make the new file outside of Quilter? Does this fix the bug?

So, I did new tests.

I created a file using Scratch with the name "test.txt" and then opened in Quilter. Using Quilter, I wrote some words and saved the same file. It worked as expected.

With the file still open, I clicked the "Save As" and decided to save to a different file named "test2". After saving, the headerbar shows "test.txt" instead, and upon checking the contents of each file, I notice that "test.txt" now has the content that was supposed to be saved as "test2", and this "test2" file has the same content.

So, file 1 is created, modified and saved as file 2. Instead of switching to file 2, and no longer writing to file 1, Quilter writes the content to both files, while keeping file 1 open.

I think the problem is in the "Save As" function. It causes file 1 to lose data when you create file 2, and does not change to file 2 as expected.
